docs/docusaurus/docs/core/define_expectations/_examples/retrieve_a_batch_of_test_data_from_a_batch_definition.py

```diff
@@ -71,6 +71,6 @@ def set_up_context_for_example(context):
 # </snippet>
 
 # <snippet name="docs/docusaurus/docs/core/define_expectations/_examples/retrieve_a_batch_of_test_data_from_a_batch_definition.py - verify populated Batch">
-batch.head()
+print(batch.head())
```

Comment:
Note for reviewer: I added `print()` so that this will do what the introductory prose promises ("printing the header and first few records of the returned Batch"). This addresses the user feedback reported in https://greatexpectations.atlassian.net/jira/software/c/projects/DOC/issues/DOC-848